crates/evm/src/evm/mod.rs

```diff
@@ -27,7 +27,7 @@ pub use primitive_types::RlpEvmTransaction;
 use sov_state::codec::BcsCodec;
 
 #[cfg(test)]
-use crate::tests::DEFAULT_CHAIN_ID;
+use crate::DEFAULT_CHAIN_ID;
 
```

Comment:
can't we have this where it used to be :)

Reply:
Done